RecoLocalCalo/HGCalRecAlgos/plugins/alpaka/HGCalRecHitCalibrationAlgorithms.dev.cc

```diff
@@ -0,0 +1,198 @@
+#include <cstddef>
+
+// CMSSW imports
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+// Alpaka imports
+#include "HeterogeneousCore/AlpakaInterface/interface/traits.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/workdivision.h"
+
+// HGCal imports
+#include "RecoLocalCalo/HGCalRecAlgos/interface/alpaka/HGCalRecHitCalibrationAlgorithms.h"
+#include "DataFormats/HGCalDigi/interface/HGCalRawDataDefinitions.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  using namespace cms::alpakatools;
+
+  //
+  struct HGCalRecHitCalibrationKernel_flagRecHits {
+    template <typename TAcc>
+    ALPAKA_FN_ACC void operator()(TAcc const& acc,
+                                  HGCalDigiDevice::View digis,
+                                  HGCalRecHitDevice::View recHits,
+                                  HGCalCalibParamDevice::ConstView calibs) const {
+      for (auto idx : uniform_elements(acc, digis.metadata().size())) {
+        auto calib = calibs[idx];
+        int calibvalid = std::to_integer<int>(calib.valid());
```

Comment:
This could be simplified to
```suggestion
        bool calibvalid = calib.valid();
```
if `calib` was changes to `uint8_t`.

Or, `calib.valid()` could actually be used directly.